GrandCentralBoard/Widgets/WebsiteAnalytics/View/HeaderCell.swift

```diff
@@ -17,26 +17,29 @@ protocol ViewDequeuing {
 final class ViewReuseController: ViewDequeuing {
     private let cellNib: UINib
     private let cellIdentifier: String
-    private let headerNib: UINib
-    private let headerIdentifier: String
+    private let headerNib: UINib?
+    private let headerIdentifier: String?
     private weak var tableView: UITableView?
 
-    init(cellNib: UINib, cellIdentifier: String, headerNib: UINib, headerIdentifier: String, tableView: UITableView) {
+    init(cellNib: UINib, cellIdentifier: String, headerNib: UINib?, headerIdentifier: String?, tableView: UITableView) {
         self.cellNib = cellNib
         self.cellIdentifier = cellIdentifier
         self.headerNib = headerNib
         self.headerIdentifier = headerIdentifier
         self.tableView = tableView
 
         tableView.registerNib(cellNib, forCellReuseIdentifier: cellIdentifier)
-        tableView.registerNib(headerNib, forHeaderFooterViewReuseIdentifier: headerIdentifier)
+        if let headerNib = headerNib, let headerIdentifier = headerIdentifier {
```

Comment:
There is a warning ⚠️   `Repeated `let` statements in conditional binding cascade should be avoided. (conditional_binding_cascade)`